### ISSUE TITLE
Request - Retexture/Items

### ISSUE THREAD

**Issue by sayblekitten** [SilentChaos512/FunOres#23]
I would like to request tiers of furnace, and grinders to make dusts. The texture pack are included in the link below.

https://drive.google.com/open?id=0B3_SUQHYCBToMEtDR0dodjB4Nlk

My idea is for 4 tiers of oven and grinder. Stone, Brick, Nether, End. Obviously, we have the stone furnace already. Ideally, the grinders would run on redstone dust as fuel, or my energized redstone dust (which is glowstone mixed with redstone, and again, has a texture, and in my ideal world stands in for/is interchangeable with yellorium and uranium)
Each furnace/grinder is twice as fast as the last. so brick is twice as fast as the default, netherbrick is twice as fast as brick, and end is twice as fast as netherbrick

**Comment by SilentChaos512** [comment]
Sorry I'm so late getting back to you on this, the past two weeks have been crazy. I do think the extra furnaces/grinders would be a good addition. They deviate from the original intentions of the mod somewhat, but I'm willing to change. Most players seem to dislike my deliberately "under-powered" machines anyway.

I was also wondering if you would be willing to redesign existing textures for the mod? None in particular, just whichever ones you see fit to change. I'll make sure you're credited, of course.

**Comment by sayblekitten** [comment]
That is a great pity that nobody else likes your machines, because 
I think your alloy smelter is brilliant, and I love the idea of nuggets 
from raw ore. But I have noticed a desire in most ppl who add mods to 
their vanilla that they want there to be way overpowered items to build 
up to, so they get cranky when it's just stuff that's has no powerup path.

     As to redesigned textures, what would you like to do? I have sadly, 
no time to learn java code, so all i have is money, as i mentioned. 
Hence, the fact that I bought the textures to fit into my ideas. What 
textures did you want to change, and what would you like to do?

     Ah, a side note. I installed funores on a 1.10 installation, and it 
loaded fine, but when i used the config file to disable monster and mob 
drop ores, the instance would crash before reaching the loading screen. 
Is that a known issue?

Eri

**Comment by SilentChaos512** [comment]
The only textures I can think of that _need_ work are the wood and stone gears. I thought you said something in your original comment (couldn't find it again for the life of me) about wanting a config option to change all the textures. Or did I get mixed up somehow?

I haven't heard about that crash. Make a new issue and upload the crash report.

**Comment by sayblekitten** [comment]
Oh, I see! I had gotten confused on what I'd asked for, and where 
we were. I don't remember saying that I wanted a config to change 
textures, but that doesn't seem like a terrible idea. My original 
request was "can you please help me assemble a mod with these ores & 
addons, stones, and these textures?" But since Funores does most all of 
what I was wanting to do, I may have asked "can you swap textures for 
mine, do you think mine are ok textures?"

     Would you also like me to commission a new texture for your alloy 
smeltery, and would you like to make it in tiers as well? I admit, that 
my mod could really use a smeltery, since some of the metals in it are 
made by alloying 2 or more items!

     I suppose I should talk a little about the original idea for my 
mod. The intent with the textures/blocks that I sent along was to create 
a 'foundation' mod, that other mods (and my own eventual grand mod idea, 
a big magic mod) could build on. So you wouldn't need to have 10 mods 
generating ores, and another 5 generating stones, and then configure 
each one of them so a world only generated one of each type. Or better 
yet, generate a vanilla world with those ores in them, and then use that 
world over and over as a jumping off point for many different kinds of 
mods. That's why I created energized redstone, it's supposed to be a 
generic "energy source", like uranium, yellorium, etc etc, but created 
with vanilla materials.

     On a continuing note, I don't really see the point of having the 
plates and gears in your mod. They're nice, i like them better than 
IC2's plates honestly, but I don't see the point. Wouldn't all the other 
mods that need gears and/or plates already have recipes that allow all 
the possible and needed gears/plates to be made?

     So, to sum up, are you asking me to get you new textures on the 
stone and wood gears? I can certainly do that, I'm more confused why you 
have gears and plates. Is there anything else you'd like retextured? I 
would actually be fine with a straight swap for my textures, leaving the 
funores existing textures and items in place.

     A thought occurs. Could I hire you to help me build the series of 
mods I was looking to make? I completely understand if you're too busy 
with your own projects, I very much understand how life can keep you busy.

Crash log has been uploaded.

Eri
